draft-ietf-ppm-dap.md

```diff
@@ -1479,36 +1487,58 @@ struct {
 } InputShareAad;
 ~~~
 
+~~~ tls-presentation
+struct {
+ReportID id;
```

Comment:
Would it be helpful in terms of efficiency to omit the ID here?  It can be inferred from the order of responses in the vector.

Reply:
Based on the "Failed upload of 1/2 reports submitted in one bulk upload" example, it seems like the intent might be that the response can include `ReportUploadStatus` values only for the reports that weren't successfully uploaded, and at that point you do need to report ID to be able to tell which it was. I kinda like that, because as we expect the vast majority of reports to be uploaded without issue, it massively cuts down on the size of responses. 

You might be able to save even more room on the wire by including an index into the vector in the `UploadRequest` instead of the report ID, but that would require clients (or that entity which uploads, which may not be a client in proxy arrangements) to maintain an index of upload requests to report IDs, which might not be tractable.